### PR TITLE
[14.0][ADD] sale_quick_blanket_order: mass line addition/update in blanket SO

### PR DESCRIPTION
**Pull request by PicchiSeba**
Adds the mass addition/update from `sale_quick` to blanket sale orders from `sale_blanket_order`

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Review by renda-dev — APPROVED** [review]
Code Review, LGTM

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers could this be merged? thanks!

**Comment by francesco-ooops** [comment]
@pedrobaeza merge please?

**Comment by francesco-ooops** [comment]
@yajo can this be merged based on existing reviews?

File: sale_quick_blanket_order/models/sale_blanket_order.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    def _get_quick_line(self, product):
+        result = self.line_ids.filtered(lambda rec: rec.product_id == product)
+
+        nr_lines = len(result.ids)
+        if nr_lines > 1:
+            raise ValidationError(
+                _(
+                    "Must have only 1 line per product for mass addition, but "
+                    "there are %s lines for the product %s"
+                    % (nr_lines, product.display_name),
```
Comment (yajo) [inline sale_quick_blanket_order/models/sale_blanket_order.py]: There's a bug here:

```suggestion
                    "Must have only 1 line per product for mass addition, but "
                    "there are %(lines)s lines for the product %(product)s",
                    lines=nr_lines,
                    product=product.display_name,
```

**Review by yajo — CHANGES_REQUESTED** [review]
OK for me, but just fix this bug I found.

**Comment by francesco-ooops** [comment]
@yajo good now? :)

**Review by yajo — APPROVED** [review]
Yes!

/ocabot merge patch

**Pull request merged**